src/core/services/fs.cpp

```diff
@@ -431,11 +432,23 @@ void FSService::deleteDirectory(u32 messagePointer) {
 	const u32 filePathPointer = mem.read32(messagePointer + 28);
 	log("FS::DeleteDirectory\n");
 
-	Helpers::warn("Stubbed FS::DeleteDirectory call!");
+	Helpers::warn("Stubbed FS::DeleteDirectory call!"); // note: should we ensure the system isn't about to delete things outside of the VFS?
```

Comment:
(This is done by isPathSafe in the archive implementations)

Reply:
Oh